packages/common/src/store/notifications/types.ts

```diff
@@ -179,6 +181,66 @@ export type RepostPushNotification = {
   }
 }
 
+export type RepostOfRepostNotification = BaseNotification & {
+  type: NotificationType.RepostOfRepost
+  entityId: ID
+  userIds: ID[]
+  entityType: Entity.Playlist | Entity.Album | Entity.Track
+}
+
+export type RepostOfRepostPushNotification = {
+  blocknumber: number
+  entityId: ID
+  initiator: ID
+  timestamp: string
+  type:
+    | PushNotificationType.RepostAlbum
+    | PushNotificationType.RepostPlaylist
+    | PushNotificationType.RepostTrack
+  actions: [
+    {
+      blocknumber: number
+      actionEntityId: ID
+      actionEntityType: Entity.User
+    }
+  ]
+  metadata: {
+    entity_owner_id: ID
+    entity_type: Entity.Album | Entity.Playlist | Entity.Track
+    entity_id: ID
+  }
+}
+
+export type FavoriteOfRepostNotification = BaseNotification & {
+  type: NotificationType.FavoriteOfRepost
+  entityId: ID
+  userIds: ID[]
+  entityType: Entity.Playlist | Entity.Album | Entity.Track
+}
+
+export type FavoriteOfRepostPushNotification = {
+  blocknumber: number
+  entityId: ID
+  initiator: ID
+  timestamp: string
+  type:
+    | PushNotificationType.RepostAlbum
```

Comment:
since this is favorite of repost, not sure if this should be `PushNotificationType.SaveAlbum` or it's own type `PushNotificationType.SaveOfFavoriteAlbum`?? or maybe it is supposed to be repost here too? i'm not sure what this type is used for so i trust whatever you decide